src/caret_analyze/value_objects/publisher.py

```diff
@@ -38,28 +55,82 @@ def __init__(
 
     @property
     def node_id(self) -> str:
+        """
+        Construct an instance.
```

Comment:
Is there a "Get node id" mistake here?

Reply:
Fix it in [8a29ff2](https://github.com/tier4/caret_analyze/pull/454/commits/8a29ff281113693de018ca4ac42f026ba7bc0e4d).